ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.WebAPI/Extensions/ValidatorRegistrations.cs

```diff
@@ -12,6 +12,7 @@ public static class ValidatorRegistrations
         public static IServiceCollection AddValidators(this IServiceCollection services)
         {
             services.AddTransient<IValidator<CreateProjectDTO>, CreateProjectDTOValidator>();
```

Comment:
you can make it Scoped as well (validator is called one time per request, so it not necessary to create validator for each calling like transient does)